docs/user-guide/compute-resources.md

```diff
@@ -159,8 +164,8 @@ The [resource quota](../admin/resource-quota.md) feature can be configured
 to limit the total amount of resources that can be consumed.  If used in conjunction
 with namespaces, it can prevent one team from hogging all the resources.
 
-### Detecting Resource Starved Containers
-To check if a container is being killed because it is hitting a resource limit, call `kubectl describe pod`
+### My container is terminated 
```

Comment:
That's the follow the advice in #9416 to organize by symptom (a you could think of it as a decision tree). I think it's a reasonable change, consistent with the "my pods are pending" topic, for instance.